content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -49,34 +49,41 @@ meta:
 
 `.dvc` files can contain the following fields:
 
-- `outs` (always present): List of <abbr>output</abbr> entries that represent
-  the files or directories tracked with DVC. Typically there is only one per
-  `.dvc` file (but several can be added or combined manually).
-- `deps`: List of <abbr>dependency</abbr> entries for this stage, only present
-  when `dvc import` and `dvc import-url` are used. Typically there is only one
-  (but several can be added manually).
-- `wdir` (optional): Working directory for the stage command to run in (relative
-  to the file's location). If this field is not present explicitly, it defaults
-  to `.` (the file's location).
+- `outs` (always present): List of <abbr>output</abbr> entries (details below)
+  that represent the files or directories tracked with DVC. Typically there is
+  only one (but several can be added or combined manually).
+- `deps`: List of <abbr>external dependency</abbr> entries (details below). Only
```

Comment:
deps in .dvc files are always external, right?

Reply:
@jorgeorpinel No, import-url works with in-repo files too. Not much product sense in that, but still.